qiskit/compiler/assembler.py

```diff
@@ -126,3 +132,151 @@ def assemble_circuits(circuits, run_config=None, qobj_header=None, qobj_id=None)
 
     return QasmQobj(qobj_id=qobj_id or str(uuid.uuid4()), config=userconfig,
                     experiments=experiments, header=qobj_header)
+
+
+def assemble_schedules(schedules, dict_config, dict_header):
+    """Assembles a list of circuits into a qobj which can be run on the backend.
+
+    Args:
+        schedules (list[ConfiguredSchedule] or ConfiguredSchedule): schedules to assemble
+        dict_config (dict): configuration of experiments
+        dict_header (dict): header to pass to the results
+
+    Returns:
+        PulseQobj: the Qobj to be run on the backends
+
+    Raises:
+        QiskitError: when invalid command is provided
+    """
+    if isinstance(schedules, ConfiguredSchedule):
+        schedules = [schedules]
+
+    experiments = []
+
+    default_qubit_lo_freq = dict_config.get('qubit_lo_freq', None)
+    default_meas_lo_freq = dict_config.get('meas_lo_freq', None)
+
+    user_pulselib = set()
+    for ii, configed in enumerate(schedules):
+        schedule = configed.schedule
+
+        # use LO frequency configs
+        lo_freqs = {}
+        if configed.config and configed.config.user_lo_dic:
+            if default_qubit_lo_freq:
+                user_qubit_los = configed.config.replaced_with_user_los(default_qubit_lo_freq)
+                if user_qubit_los != default_qubit_lo_freq:
+                    lo_freqs['qubit_lo_freq'] = user_qubit_los
+            if default_meas_lo_freq:
+                user_meas_los = configed.config.replaced_with_user_los(default_meas_lo_freq)
+                if user_meas_los != default_meas_lo_freq:
+                    lo_freqs['meas_lo_freq'] = user_meas_los
+
+        # generate experimental configuration
+        experimentconfig = PulseQobjExperimentConfig(**lo_freqs)
+
+        # generate experimental header
+        experimentheader = QobjExperimentHeader(name=configed.name or 'Experiment-%d' % ii)
+
+        commands = []
+        for block in schedule.flat_instruction_sequence():
+            pulse_instr = block.instruction
+            if isinstance(pulse_instr, DriveInstruction):
```

Comment:
Could you please break these into separate method calls? I think this would make it easier to understand at a glance. 

It might even make sense to have these as a method for a `Command`, `to_qobj`. Which would allow users to create their custom `Command`s and submit them to the backend. However, I think this fails for the case of the acquire instruction as the acquire requires information about all other acquires at that time. I think this is more a failing of how the spec defines acquires then anything else.

Reply:
`to_qobj` sounds nice, but I want import qobj models only in `assembler` as circuit is.

I believe it works.
```
current_commands  = PulseQobjInstruction(**pulse_instr.to_dict())
```

`AcquireInstruction` has a list of qubit and memory/register slots, and we can generate qobj instruction from a single command instruction. Violation of meas_map can be checked by another validator.

Reply:
I guess what I mean is how do we enforce that all acquires are present in a measurement map for our current examples? Are you just making sure you add acquires for every qubit?

Reply:
I believe schedule is independent of device constraints, it should be able to create arbitrary sequence. All violations of the backend will be evaluated before it goes to the backend.
Another idea is to change `qubit` argumanent of `AcquisitionInstruction` to `meas_map` itself. `DeviceSpecification` returns a list of `meas_map` and we specify qubits by a unit of `meas_map`. In this case, we can enforce a user to align with `meas_map`.